_includes/ee-linux-install-reuse.md

```diff
@@ -134,8 +134,8 @@ You can install Docker EE in different ways, depending on your needs:
     $ sudo yum -y install docker-ee
     ```
 
-    If this is the first time you have refreshed the package index since adding
-    the Docker repositories, you will be prompted to accept the GPG key, and
+    If this is the first time you are installing a package from a recently added
+    repository, you will be prompted to accept the GPG key, and
```

Comment:
I'm not sure this change really adds much.

Reply:
The reason is, originally this text was right after `yum makecache` (which refreshed the package index), and now it is (correctly) moved to after `yum install`. Also, refreshing the package index doesn't really lead to asking a user to accept a GPG key, so the statement is just incorrect.